clients/bucket/delete_test.go

```diff
@@ -3,6 +3,7 @@ package bucket_test
 import (
 	"bytes"
 	"context"
+	"github.com/influxdata/influx-cli/v2/clients"
```

Comment:
Bad import sort (I'd expect `make fmt` to fix it, let me know if not)

Reply:
Nope, `make fmt` was already run on this.